src/System.Private.CoreLib/shared/System/Runtime/AmbiguousImplementationException.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime;
+using System.Globalization;
+
+namespace System.Runtime.Serialization
+{
+    [Serializable]
+    public sealed class AmbiguousImplementationException : Exception
+    {
+        public AmbiguousImplementationException()
+            : base(SR.AmbiguousImplementationException_NullMessage)
+        {
+        }
+
+        public AmbiguousImplementationException(string methodName, string interfaceName, string typeName, string assemblyName)
```

Comment:
Where did this ctor come from?  I don't see it in the issue.

Reply:
The error string proposed in the issue has parameters, so I created this ctor to populate them. https://github.com/dotnet/corefx/issues/34124#issuecomment-453100548

Reply:
That's not a good reason to make the constructor public.

This error message is likely going to be formatted in the unmanaged runtime anyway, without using this constructor.

Reply:
> Should I remove the string from the resx as well?

Yes, thanks.

Reply:
Makes sense. Should I do something different with the parameterless constructor, or is the string I came up for that fine?

Reply:
I would make the null message more generic, e.g. `Ambiguous implementation found.`.